docs/concepts/projects.md

```diff
@@ -71,6 +71,14 @@ build and install the project into the project environment. By default, projects
 [editable mode](https://setuptools.pypa.io/en/latest/userguide/development_mode.html) so changes to
 the source code are reflected immediately, without re-installation.
 
+To define uv as build backend:
+
+```toml
+[build-system]
+requires = ["uv>=0.4.18,<5"]
```

Comment:
This is a really large range. :)

Reply:
Also, you only need an upper bound if there's custom configuration that might change. `[project]` is fine, it's just if there are `tool.uv.*` options that might change in the future. I don't see any custom configuration explained at all below?

Reply:
FWIW, scikit-build-core handles this by having the minimum version be something a user can specify, and then the any changed behaviors will keep the same behavior as the minimum version specified. (CMake works the same way)

So if we change a default, or a name, we respect the old way of doing things if the minimum version is set and below the value where it changed.

Reply:
We will add custom options, we need them for inclusions and exclusions, and from my experience in maturin and ruff we need to be able to evolve them.